examples/bmp280/bmp280_example.c

```diff
@@ -6,66 +6,89 @@
 #include "FreeRTOS.h"
 #include "task.h"
 
+#include "i2c/i2c.h"
 #include "bmp280/bmp280.h"
 
 // In forced mode user initiate measurement each time.
 // In normal mode measurement is done continuously with specified standby time. 
 // #define MODE_FORCED
 
-const uint8_t scl_pin = 5;
-const uint8_t sda_pin = 4;
+const uint8_t scl_pin = 0;
+const uint8_t sda_pin = 2;
 
 #ifdef MODE_FORCED
 static void bmp280_task_forced(void *pvParameters)
 {
     bmp280_params_t  params;
-    float pressure, temperature;
+    float pressure, temperature, humidity;
 
     bmp280_init_default_params(&params);
     params.mode = BMP280_MODE_FORCED;
 
+    bmp280_calib_t bmp280_calib;
+    bmp280_calib.i2c_addr = BMP280_I2C_ADDRESS_0;
```

Comment:
`bmp280_calib_t` now becomes something like a class instance in OOP terms. It's no longer just a calibration data. It might be better called something like `bmp280_t` and have all the instance variables inside including calibration data. And I don't really like to expose calibration data to a user as the user doesn't care about it, so it's better to make it "private". But it's a cosmetic comments and might be a separate PR.